pkg/kvstore/allocator/allocator.go

```diff
@@ -444,6 +453,11 @@ func (k *kvstoreBackend) RunGC(ctx context.Context, rateLimit *rate.Limiter, sta
 		return nil, fmt.Errorf("list failed: %s", err)
 	}
 
+	gcResult := gcStats{
+		alive:   len(allocated),
```

Comment:
Here it seems that we never update the `alive` identities. I think that alive here means the total identities allocated? Or maybe we should subtract the number of deleted identities before populating the metric.

Reply:
Nice point! I forgot to subtract the deleted identities.